ConwaysGameOfLife/InputParser.cs

```diff
@@ -44,7 +44,5 @@ private static int ParseInput(string numberInput)
             }
             return -1;

```

Comment:
Do we need this wrapper for `Int32.TryParse`? We lose a bit of functionality here, given that if we pass in "-1" vs an invalid `int`, we can't tell the outputs apart.